2022-12-13 12:27:54.783[1670930874783] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:27:56.404[1670930876404] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:27:57.687[1670930877687] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:27:57.697[1670930877697] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:27:57.700[1670930877700] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:27:57.702[1670930877702] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:27:57.791[1670930877791] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:27:58.200[1670930878200] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:27:58.310[1670930878310] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:27:58.343[1670930878343] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@9d4dcd60 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@745e5390 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat34aci8dypkfe|35e478f, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@13f0a3c0 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat34aci8dypkfe|27f0ad19, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat34aci8dypkfe|63429932, numHelperThreads -> 3 ]
2022-12-13 12:27:59.892[1670930879892] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:28:03.078[1670930883078] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@3a7e365] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:28:03.430[1670930883430] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:35:05.142[1670931305142] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:35:06.508[1670931306508] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:35:06.988[1670931306988] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:35:06.994[1670931306994] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:35:06.995[1670931306995] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:35:06.995[1670931306995] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:35:07.051[1670931307051] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:35:07.408[1670931307408] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:35:07.518[1670931307518] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:35:07.554[1670931307554] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@cfb45ade [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@b6b8dfb7 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat34jjpb13e4hic|6e28bb87, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@4d9c6132 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat34jjpb13e4hic|4a8b5227, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat34jjpb13e4hic|726a6b94, numHelperThreads -> 3 ]
2022-12-13 12:35:08.865[1670931308865] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:35:10.326[1670931310326] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@798dad3d] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:35:10.499[1670931310499] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:35:10.974[1670931310974] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1146, SQLState: 42S02
2022-12-13 12:35:10.974[1670931310974] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Table 'test.symbols' doesn't exist
2022-12-13 12:35:11.025[1670931311025] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value
java.sql.SQLSyntaxErrorException: Table 'test.binance_data_seq' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1009)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:1471)
	at org.hibernate.id.enhanced.TableStructure.executeQuery(TableStructure.java:228)
	at org.hibernate.id.enhanced.TableStructure.access$300(TableStructure.java:45)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:150)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:138)
	at org.hibernate.jdbc.WorkExecutor.executeReturningWork(WorkExecutor.java:55)
	at org.hibernate.jdbc.AbstractReturningWork.accept(AbstractReturningWork.java:34)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcIsolationDelegate.delegateWork(JdbcIsolationDelegate.java:57)
	at org.hibernate.id.enhanced.TableStructure$1.getNextValue(TableStructure.java:137)
	at org.hibernate.id.enhanced.PooledOptimizer.generate(PooledOptimizer.java:73)
	at org.hibernate.id.enhanced.SequenceStyleGenerator.generate(SequenceStyleGenerator.java:534)
	at org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:115)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:194)
	at org.hibernate.event.internal.DefaultSaveEventListener.saveWithGeneratedOrRequestedId(DefaultSaveEventListener.java:38)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:179)
	at org.hibernate.event.internal.DefaultSaveEventListener.performSaveOrUpdate(DefaultSaveEventListener.java:32)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:75)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107)
	at org.hibernate.internal.SessionImpl.fireSave(SessionImpl.java:672)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:665)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:660)
	at persistence.DbWriter.write(DbWriter.java:104)
	at persistence.DbWriterTest2.canAdd_oneBinanceData(DbWriterTest2.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-12-13 12:35:11.027[1670931311027] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1146, SQLState: 42S02
2022-12-13 12:35:11.027[1670931311027] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Table 'test.binance_data_seq' doesn't exist
2022-12-13 12:44:59.859[1670931899859] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:45:01.359[1670931901359] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:45:01.831[1670931901831] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:45:01.838[1670931901838] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:45:01.839[1670931901839] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:45:01.839[1670931901839] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:45:01.903[1670931901903] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:45:02.255[1670931902255] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:45:02.364[1670931902364] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:45:02.401[1670931902401] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@d674e7fe [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@12468494 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat34wap41lhvi8v|17d238b1, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@cd6421f4 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat34wap41lhvi8v|315ba14a, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat34wap41lhvi8v|7c8326a4, numHelperThreads -> 3 ]
2022-12-13 12:45:03.627[1670931903627] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:45:05.414[1670931905414] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@7d986d83] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:45:05.979[1670931905979] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:45:06.609[1670931906609] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value
java.sql.SQLSyntaxErrorException: Table 'test.binance_data_seq' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1009)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:1471)
	at org.hibernate.id.enhanced.TableStructure.executeQuery(TableStructure.java:228)
	at org.hibernate.id.enhanced.TableStructure.access$300(TableStructure.java:45)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:150)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:138)
	at org.hibernate.jdbc.WorkExecutor.executeReturningWork(WorkExecutor.java:55)
	at org.hibernate.jdbc.AbstractReturningWork.accept(AbstractReturningWork.java:34)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcIsolationDelegate.delegateWork(JdbcIsolationDelegate.java:57)
	at org.hibernate.id.enhanced.TableStructure$1.getNextValue(TableStructure.java:137)
	at org.hibernate.id.enhanced.PooledOptimizer.generate(PooledOptimizer.java:73)
	at org.hibernate.id.enhanced.SequenceStyleGenerator.generate(SequenceStyleGenerator.java:534)
	at org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:115)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:194)
	at org.hibernate.event.internal.DefaultSaveEventListener.saveWithGeneratedOrRequestedId(DefaultSaveEventListener.java:38)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:179)
	at org.hibernate.event.internal.DefaultSaveEventListener.performSaveOrUpdate(DefaultSaveEventListener.java:32)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:75)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107)
	at org.hibernate.internal.SessionImpl.fireSave(SessionImpl.java:672)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:665)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:660)
	at persistence.DbWriter.write(DbWriter.java:104)
	at persistence.DbWriterTest2.canAdd_oneBinanceData(DbWriterTest2.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-12-13 12:45:06.614[1670931906614] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1146, SQLState: 42S02
2022-12-13 12:45:06.617[1670931906617] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Table 'test.binance_data_seq' doesn't exist
2022-12-13 12:45:26.060[1670931926060] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:45:27.853[1670931927853] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:45:28.357[1670931928357] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:45:28.363[1670931928363] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:45:28.364[1670931928364] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:45:28.364[1670931928364] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:45:28.431[1670931928431] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:45:28.787[1670931928787] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:45:28.898[1670931928898] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:45:28.935[1670931928935] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@3a0b77d0 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@75dd1466 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat34wv5n4x10tm|17d238b1, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@30fab1c6 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat34wv5n4x10tm|315ba14a, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat34wv5n4x10tm|7c8326a4, numHelperThreads -> 3 ]
2022-12-13 12:45:30.185[1670931930185] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:45:32.448[1670931932448] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@7d986d83] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:45:32.946[1670931932946] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:45:33.786[1670931933786] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value
java.sql.SQLSyntaxErrorException: Table 'test.binance_data_seq' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1009)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:1471)
	at org.hibernate.id.enhanced.TableStructure.executeQuery(TableStructure.java:228)
	at org.hibernate.id.enhanced.TableStructure.access$300(TableStructure.java:45)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:150)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:138)
	at org.hibernate.jdbc.WorkExecutor.executeReturningWork(WorkExecutor.java:55)
	at org.hibernate.jdbc.AbstractReturningWork.accept(AbstractReturningWork.java:34)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcIsolationDelegate.delegateWork(JdbcIsolationDelegate.java:57)
	at org.hibernate.id.enhanced.TableStructure$1.getNextValue(TableStructure.java:137)
	at org.hibernate.id.enhanced.PooledOptimizer.generate(PooledOptimizer.java:73)
	at org.hibernate.id.enhanced.SequenceStyleGenerator.generate(SequenceStyleGenerator.java:534)
	at org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:115)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:194)
	at org.hibernate.event.internal.DefaultSaveEventListener.saveWithGeneratedOrRequestedId(DefaultSaveEventListener.java:38)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:179)
	at org.hibernate.event.internal.DefaultSaveEventListener.performSaveOrUpdate(DefaultSaveEventListener.java:32)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:75)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107)
	at org.hibernate.internal.SessionImpl.fireSave(SessionImpl.java:672)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:665)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:660)
	at persistence.DbWriter.write(DbWriter.java:104)
	at persistence.DbWriterTest2.canAdd_oneBinanceData(DbWriterTest2.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-12-13 12:45:33.789[1670931933789] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1146, SQLState: 42S02
2022-12-13 12:45:33.789[1670931933789] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Table 'test.binance_data_seq' doesn't exist
2022-12-13 12:46:50.211[1670932010211] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:46:51.791[1670932011791] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:46:52.274[1670932012274] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:46:52.280[1670932012280] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:46:52.281[1670932012281] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:46:52.281[1670932012281] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:46:52.338[1670932012338] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:46:52.678[1670932012678] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:46:52.780[1670932012780] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:46:52.815[1670932012815] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@12cb3e6f [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@c805fc72 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat34ynwe1vp12j9|757194dc, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@2ebbfd34 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat34ynwe1vp12j9|aa5455e, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat34ynwe1vp12j9|69b2f8e5, numHelperThreads -> 3 ]
2022-12-13 12:46:54.165[1670932014165] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:46:55.843[1670932015843] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@a09303] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:46:56.308[1670932016308] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:46:56.730[1670932016730] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1146, SQLState: 42S02
2022-12-13 12:46:56.731[1670932016731] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Table 'test.symbols' doesn't exist
2022-12-13 12:46:56.767[1670932016767] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value
java.sql.SQLSyntaxErrorException: Table 'test.binance_data_seq' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1009)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:1471)
	at org.hibernate.id.enhanced.TableStructure.executeQuery(TableStructure.java:228)
	at org.hibernate.id.enhanced.TableStructure.access$300(TableStructure.java:45)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:150)
	at org.hibernate.id.enhanced.TableStructure$1$1.execute(TableStructure.java:138)
	at org.hibernate.jdbc.WorkExecutor.executeReturningWork(WorkExecutor.java:55)
	at org.hibernate.jdbc.AbstractReturningWork.accept(AbstractReturningWork.java:34)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcIsolationDelegate.delegateWork(JdbcIsolationDelegate.java:57)
	at org.hibernate.id.enhanced.TableStructure$1.getNextValue(TableStructure.java:137)
	at org.hibernate.id.enhanced.PooledOptimizer.generate(PooledOptimizer.java:73)
	at org.hibernate.id.enhanced.SequenceStyleGenerator.generate(SequenceStyleGenerator.java:534)
	at org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:115)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:194)
	at org.hibernate.event.internal.DefaultSaveEventListener.saveWithGeneratedOrRequestedId(DefaultSaveEventListener.java:38)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:179)
	at org.hibernate.event.internal.DefaultSaveEventListener.performSaveOrUpdate(DefaultSaveEventListener.java:32)
	at org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:75)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107)
	at org.hibernate.internal.SessionImpl.fireSave(SessionImpl.java:672)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:665)
	at org.hibernate.internal.SessionImpl.save(SessionImpl.java:660)
	at persistence.DbWriter.write(DbWriter.java:104)
	at persistence.DbWriterTest2.canAdd_oneBinanceData(DbWriterTest2.java:38)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-12-13 12:46:56.769[1670932016769] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1146, SQLState: 42S02
2022-12-13 12:46:56.769[1670932016769] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Table 'test.binance_data_seq' doesn't exist
2022-12-13 12:48:59.869[1670932139869] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:49:01.228[1670932141228] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:49:01.693[1670932141693] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:49:01.699[1670932141699] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:49:01.700[1670932141700] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:49:01.700[1670932141700] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:49:01.765[1670932141765] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:49:02.098[1670932142098] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:49:02.201[1670932142201] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:49:02.232[1670932142232] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@a7160184 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@5c50bf87 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat351frnfqqurp|757194dc, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@e932037f [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat351frnfqqurp|aa5455e, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat351frnfqqurp|69b2f8e5, numHelperThreads -> 3 ]
2022-12-13 12:49:03.539[1670932143539] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:49:05.109[1670932145109] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@a09303] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:49:05.616[1670932145616] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:49:06.008[1670932146008] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1701, SQLState: 42000
2022-12-13 12:49:06.008[1670932146008] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Cannot truncate a table referenced in a foreign key constraint (`test`.`binance_data`, CONSTRAINT `FKduh1own3cqfv6jwe9255tcjgf`)
2022-12-13 12:49:29.608[1670932169608] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:49:30.977[1670932170977] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:49:31.405[1670932171405] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:49:31.411[1670932171411] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:49:31.412[1670932171412] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:49:31.412[1670932171412] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:49:31.472[1670932171472] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:49:31.800[1670932171800] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:49:31.902[1670932171902] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:49:31.938[1670932171938] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@be4d5d6d [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@9e5824bc [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat3522oxqby1ud|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@ddc23173 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat3522oxqby1ud|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat3522oxqby1ud|522b2631, numHelperThreads -> 3 ]
2022-12-13 12:49:33.160[1670932173160] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:49:34.470[1670932174470] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:49:34.633[1670932174633] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:49:34.977[1670932174977] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1701, SQLState: 42000
2022-12-13 12:49:34.977[1670932174977] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Cannot truncate a table referenced in a foreign key constraint (`test`.`binance_data`, CONSTRAINT `FKduh1own3cqfv6jwe9255tcjgf`)
2022-12-13 12:49:35.120[1670932175120] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1701, SQLState: 42000
2022-12-13 12:49:35.120[1670932175120] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - Cannot truncate a table referenced in a foreign key constraint (`test`.`binance_data`, CONSTRAINT `FKduh1own3cqfv6jwe9255tcjgf`)
2022-12-13 12:51:54.210[1670932314210] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:51:55.670[1670932315670] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:51:56.201[1670932316201] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:51:56.207[1670932316207] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:51:56.208[1670932316208] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:51:56.208[1670932316208] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:51:56.268[1670932316268] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:51:56.682[1670932316682] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:51:56.799[1670932316799] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:51:56.840[1670932316840] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@3e41963c [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@1e4c5d8b [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat3556f9xwbmyk|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@5db66a42 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat3556f9xwbmyk|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat3556f9xwbmyk|522b2631, numHelperThreads -> 3 ]
2022-12-13 12:51:58.157[1670932318157] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:51:59.452[1670932319452] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:51:59.620[1670932319620] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:51:59.961[1670932319961] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 12:51:59.962[1670932319962] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'truncate TABLE symbols' at line 1
2022-12-13 12:52:00.100[1670932320100] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 12:52:00.101[1670932320101] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'truncate TABLE symbols' at line 1
2022-12-13 12:52:23.383[1670932343383] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:52:24.776[1670932344776] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:52:25.231[1670932345231] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:52:25.238[1670932345238] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:52:25.238[1670932345238] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:52:25.239[1670932345239] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:52:25.293[1670932345293] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:52:25.657[1670932345657] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:52:25.774[1670932345774] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:52:25.812[1670932345812] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@4ab04681 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@2abb0dd0 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat355st45y59d3|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@6a251a87 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat355st45y59d3|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat355st45y59d3|522b2631, numHelperThreads -> 3 ]
2022-12-13 12:52:27.349[1670932347349] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:52:28.913[1670932348913] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:52:29.065[1670932349065] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:52:29.842[1670932349842] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 12:52:29.842[1670932349842] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'truncate TABLE symbols' at line 1
2022-12-13 12:52:30.182[1670932350182] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 12:52:30.182[1670932350182] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'truncate TABLE symbols' at line 1
2022-12-13 12:53:53.473[1670932433473] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:53:54.923[1670932434923] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:53:55.384[1670932435384] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:53:55.390[1670932435390] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:53:55.391[1670932435391] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:53:55.391[1670932435391] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:53:55.450[1670932435450] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:53:55.790[1670932435790] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:53:55.906[1670932435906] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:53:55.945[1670932435945] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@d9aa7330 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@b9b53a7f [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat357qdk1n1a13d|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@f91f4736 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat357qdk1n1a13d|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat357qdk1n1a13d|522b2631, numHelperThreads -> 3 ]
2022-12-13 12:53:57.413[1670932437413] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:53:58.857[1670932438857] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:53:59.031[1670932439031] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:53:59.459[1670932439459] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 12:53:59.460[1670932439460] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'truncate binance_data_seq' at line 1
2022-12-13 12:53:59.679[1670932439679] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 12:53:59.679[1670932439679] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'truncate binance_data_seq' at line 1
2022-12-13 12:54:45.965[1670932485965] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:54:47.652[1670932487652] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:54:48.135[1670932488135] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:54:48.141[1670932488141] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:54:48.141[1670932488141] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:54:48.142[1670932488142] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:54:48.202[1670932488202] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:54:48.547[1670932488547] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:54:48.658[1670932488658] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:54:48.698[1670932488698] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@7d5de1c6 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@5d68a915 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat358v2u134ulbd|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@9cd2b5cc [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat358v2u134ulbd|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat358v2u134ulbd|522b2631, numHelperThreads -> 3 ]
2022-12-13 12:54:50.075[1670932490075] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:54:51.649[1670932491649] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:54:51.800[1670932491800] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:54:53.201[1670932493201] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 12:57:43.827[1670932663827] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 12:57:45.226[1670932665226] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 12:57:45.693[1670932665693] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 12:57:45.700[1670932665700] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 12:57:45.701[1670932665701] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 12:57:45.701[1670932665701] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 12:57:45.758[1670932665758] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 12:57:46.099[1670932666099] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 12:57:46.206[1670932666206] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 12:57:46.244[1670932666244] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@19a44ba [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@e89ec993 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat35co361k0yhms|6e28bb87, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@7f824b0e [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat35co361k0yhms|4a8b5227, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat35co361k0yhms|726a6b94, numHelperThreads -> 3 ]
2022-12-13 12:57:47.659[1670932667659] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 12:57:49.210[1670932669210] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@798dad3d] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 12:57:49.376[1670932669376] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 12:57:50.728[1670932670728] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:06:05.740[1670933165740] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:06:07.225[1670933167225] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:06:07.728[1670933167728] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:06:07.735[1670933167735] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:06:07.736[1670933167736] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:06:07.736[1670933167736] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:06:07.795[1670933167795] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:06:08.135[1670933168135] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:06:08.244[1670933168244] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:06:08.279[1670933168279] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@ab9fc87 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@eac4c3d6 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat35nfgc1sffmey|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@2a2ed08d [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat35nfgc1sffmey|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat35nfgc1sffmey|522b2631, numHelperThreads -> 3 ]
2022-12-13 13:06:09.675[1670933169675] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:06:11.160[1670933171160] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:06:11.313[1670933171313] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:06:11.758[1670933171758] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 13:06:11.759[1670933171759] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'binance_data_seq' at line 1
2022-12-13 13:06:11.979[1670933171979] | WARN  | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - SQL Error: 1064, SQLState: 42000
2022-12-13 13:06:11.980[1670933171980] | ERROR | main       | o.h.e.jdbc.spi.SqlExceptionHelper    - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'binance_data_seq' at line 1
2022-12-13 13:06:34.633[1670933194633] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:06:36.051[1670933196051] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:06:36.516[1670933196516] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:06:36.523[1670933196523] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:06:36.524[1670933196524] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:06:36.524[1670933196524] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:06:36.583[1670933196583] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:06:37.013[1670933197013] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:06:37.120[1670933197120] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:06:37.152[1670933197152] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@9da3afb3 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@7dae7702 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat35o1op1xedk9n|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@bd1883b9 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat35o1op1xedk9n|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat35o1op1xedk9n|522b2631, numHelperThreads -> 3 ]
2022-12-13 13:06:38.481[1670933198481] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:06:40.221[1670933200221] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:06:40.384[1670933200384] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:06:41.405[1670933201405] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:08:02.828[1670933282828] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:08:04.212[1670933284212] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:08:04.740[1670933284740] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:08:04.747[1670933284747] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:08:04.748[1670933284748] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:08:04.748[1670933284748] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:08:04.812[1670933284812] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:08:05.177[1670933285177] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:08:05.344[1670933285344] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:08:05.392[1670933285392] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@cb688bf9 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@88019690 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat35pxqt11mlbr6|5d05ef57, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@48b8028 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat35pxqt11mlbr6|273c947f, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat35pxqt11mlbr6|61533ae, numHelperThreads -> 3 ]
2022-12-13 13:08:06.769[1670933286769] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:08:08.152[1670933288152] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4bdf] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:08:08.306[1670933288306] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:08:08.724[1670933288724] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:08:25.384[1670933305384] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:08:27.284[1670933307284] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:08:28.017[1670933308017] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:08:28.025[1670933308025] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:08:28.026[1670933308026] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:08:28.026[1670933308026] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:08:28.098[1670933308098] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:08:28.628[1670933308628] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:08:28.734[1670933308734] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:08:28.775[1670933308775] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@c613a407 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@e2b98042 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat35qfq1ohgr2w|5d05ef57, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@5f4369da [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat35qfq1ohgr2w|273c947f, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat35qfq1ohgr2w|61533ae, numHelperThreads -> 3 ]
2022-12-13 13:08:30.216[1670933310216] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:08:31.843[1670933311843] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4bdf] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:08:32.078[1670933312078] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:38:56.490[1670935136490] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:38:57.986[1670935137986] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:38:58.472[1670935138472] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:38:58.472[1670935138472] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:38:58.472[1670935138472] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:38:58.472[1670935138472] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:38:58.541[1670935138541] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:38:58.874[1670935138874] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:38:58.974[1670935138974] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:38:59.021[1670935139021] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@2e6ab88c [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@e2a3daf8 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat36to3c1gowoya|49096b06, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@fd59f1ed [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat36to3c1gowoya|50cf5a23, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat36to3c1gowoya|d02f8d, numHelperThreads -> 3 ]
2022-12-13 13:39:00.415[1670935140415] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:39:02.024[1670935142024] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@3f6bf8aa] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:39:02.187[1670935142187] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:39:02.720[1670935142720] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:39:26.979[1670935166979] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:39:28.372[1670935168372] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:39:28.836[1670935168836] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:39:28.836[1670935168836] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:39:28.836[1670935168836] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:39:28.836[1670935168836] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:39:28.898[1670935168898] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:39:29.236[1670935169236] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:39:29.352[1670935169352] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:39:29.383[1670935169383] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@5501208b [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@317b943e [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat36ubja1moxv64|5d05ef57, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@ae057dd6 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat36ubja1moxv64|273c947f, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat36ubja1moxv64|61533ae, numHelperThreads -> 3 ]
2022-12-13 13:39:30.648[1670935170648] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:39:32.107[1670935172107] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4bdf] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:39:32.547[1670935172547] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:39:33.095[1670935173095] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:40:02.416[1670935202416] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:40:03.765[1670935203765] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:40:04.229[1670935204229] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:40:04.238[1670935204238] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:40:04.238[1670935204238] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:40:04.238[1670935204238] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:40:04.299[1670935204299] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:40:04.627[1670935204627] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:40:04.727[1670935204727] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:40:04.765[1670935204765] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@13e8ec73 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@834837e6 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat36v2u71cqwkac|213deac2, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@51439e72 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat36v2u71cqwkac|30457e14, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat36v2u71cqwkac|732bb66d, numHelperThreads -> 3 ]
2022-12-13 13:40:06.093[1670935206093] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:40:07.443[1670935207443] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4cad79bc] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:40:07.612[1670935207612] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:40:08.146[1670935208146] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:41:19.867[1670935279867] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:41:21.626[1670935281626] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:41:22.119[1670935282119] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:41:22.135[1670935282135] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:41:22.135[1670935282135] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:41:22.135[1670935282135] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:41:22.204[1670935282204] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:41:22.505[1670935282505] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:41:22.620[1670935282620] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:41:22.667[1670935282667] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@5e4b27b6 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@51727273 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat36wqy3kn705s|5d05ef57, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@cdfc5c0b [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat36wqy3kn705s|273c947f, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat36wqy3kn705s|61533ae, numHelperThreads -> 3 ]
2022-12-13 13:41:24.030[1670935284030] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:41:25.414[1670935285414] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4bdf] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:41:25.862[1670935285862] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:41:26.348[1670935286348] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:42:09.794[1670935329794] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:42:11.172[1670935331172] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:42:11.679[1670935331679] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:42:11.679[1670935331679] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:42:11.679[1670935331679] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:42:11.679[1670935331679] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:42:11.751[1670935331751] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:42:12.136[1670935332136] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:42:12.252[1670935332252] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:42:12.306[1670935332306] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@272179ff [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@a366694a [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat36xt6k5txr0k|5d05ef57, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@1ff052e2 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat36xt6k5txr0k|273c947f, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat36xt6k5txr0k|61533ae, numHelperThreads -> 3 ]
2022-12-13 13:42:13.825[1670935333825] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:42:15.741[1670935335741] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4bdf] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:42:15.910[1670935335910] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:42:16.411[1670935336411] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:43:57.436[1670935437436] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:43:58.853[1670935438853] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:43:59.342[1670935439342] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:43:59.358[1670935439358] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:43:59.358[1670935439358] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:43:59.358[1670935439358] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:43:59.405[1670935439405] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:43:59.749[1670935439749] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:43:59.865[1670935439865] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:43:59.896[1670935439896] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@d57ebaa2 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@c0b13d07 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat37048xjaj9md|5d05ef57, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@3d3b269f [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat37048xjaj9md|273c947f, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat37048xjaj9md|61533ae, numHelperThreads -> 3 ]
2022-12-13 13:44:01.199[1670935441199] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:44:02.758[1670935442758] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@4bdf] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:44:02.905[1670935442905] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:44:03.422[1670935443422] | ERROR | main       | o.h.id.enhanced.TableStructure       - could not read a hi value - you need to populate the table: binance_data_seq
2022-12-13 13:45:28.929[1670935528929] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:45:30.222[1670935530222] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:45:30.570[1670935530570] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:45:30.586[1670935530586] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:45:30.586[1670935530586] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:45:30.586[1670935530586] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:45:30.639[1670935530639] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:45:31.002[1670935531002] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:45:31.118[1670935531118] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:45:31.140[1670935531140] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@5f65b335 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@c0cacf61 [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat3722n01bh0ypj|49096b06, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@db80e656 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat3722n01bh0ypj|50cf5a23, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat3722n01bh0ypj|d02f8d, numHelperThreads -> 3 ]
2022-12-13 13:45:32.368[1670935532368] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:45:33.597[1670935533597] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@3f6bf8aa] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:45:33.760[1670935533760] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:45:34.299[1670935534299] | INFO  | main       | o.h.id.enhanced.PooledOptimizer      - HHH000201: Pooled optimizer source reported [BasicHolder[java.lang.Long[0]]] as the initial value; use of 1 or greater highly recommended
2022-12-13 13:45:53.024[1670935553024] | INFO  | main       | org.hibernate.Version                - HHH000412: Hibernate ORM core version 5.6.8.Final
2022-12-13 13:45:54.392[1670935554392] | INFO  | main       | o.h.annotations.common.Version       - HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2022-12-13 13:45:54.931[1670935554931] | INFO  | main       | o.h.e.j.c.i.ConnectionProviderInitiator - HHH000130: Instantiating explicit connection provider: org.hibernate.connection.C3P0ConnectionProvider
2022-12-13 13:45:54.942[1670935554942] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH010002: C3P0 using driver: com.mysql.cj.jdbc.Driver at URL: jdbc:mysql://localhost:3306/test
2022-12-13 13:45:54.942[1670935554942] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001001: Connection properties: {password=****, user=root}
2022-12-13 13:45:54.942[1670935554942] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001003: Autocommit mode: false
2022-12-13 13:45:55.022[1670935555022] | INFO  | MLog-Init-Reporter | com.mchange.v2.log.MLog              - MLog clients using slf4j logging.
2022-12-13 13:45:55.406[1670935555406] | INFO  | main       | com.mchange.v2.c3p0.C3P0Registry     - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2022-12-13 13:45:55.532[1670935555532] | INFO  | main       | o.h.c.i.C3P0ConnectionProvider       - HHH10001007: JDBC isolation level: <unknown>
2022-12-13 13:45:55.567[1670935555567] | INFO  | main       | c.m.v.c.i.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.PoolBackedDataSource@610eadc0 [ connectionPoolDataSource -> com.mchange.v2.c3p0.WrapperConnectionPoolDataSource@4119750f [ acquireIncrement -> 1, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, debugUnreturnedConnectionStackTraces -> false, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, identityToken -> 1hgengqat372lgz16db776|72ab05ed, idleConnectionTestPeriod -> 3000, initialPoolSize -> 10, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 50, maxStatementsPerConnection -> 0, minPoolSize -> 10, nestedDataSource -> com.mchange.v2.c3p0.DriverManagerDataSource@808381c6 [ description -> null, driverClass -> null, factoryClassLocation -> null, forceUseNamedDriverClass -> false, identityToken -> 1hgengqat372lgz16db776|6979efad, jdbcUrl -> jdbc:mysql://localhost:3306/test, properties -> {password=******, user=******} ], preferredTestQuery -> null, privilegeSpawnedThreads -> false, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, usesTraditionalReflectiveProxies -> false; userOverrides: {} ], dataSourceName -> null, extensions -> {}, factoryClassLocation -> null, identityToken -> 1hgengqat372lgz16db776|522b2631, numHelperThreads -> 3 ]
2022-12-13 13:45:57.604[1670935557604] | INFO  | main       | org.hibernate.dialect.Dialect        - HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2022-12-13 13:46:00.180[1670935560180] | INFO  | main       | org.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@696db620] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2022-12-13 13:46:00.435[1670935560435] | INFO  | main       | o.h.e.t.j.p.i.JtaPlatformInitiator   - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-12-13 13:46:01.826[1670935561826] | INFO  | main       | o.h.id.enhanced.PooledOptimizer      - HHH000201: Pooled optimizer source reported [BasicHolder[java.lang.Long[0]]] as the initial value; use of 1 or greater highly recommended
